aten/src/ATen/native/README.md

```diff
@@ -61,8 +61,7 @@ signature.
 - `int`. Think about this like a Python int. This is translated into a C++ argument of type `int64_t`.
 - `float`. Think about this like a Python `float`. It is translated into a C++ argument of type `double`.
 - `bool`
-- `str`.  It is translated into a C++ argument of type `std::string`
-  (but we should fix this, see https://github.com/pytorch/pytorch/issues/53546)
+- `str`.  It is translated into a C++ argument of type `c10::string_view`
```

Comment:
I think we should also mention that c10::string_view is non-owning

Reply:
Ah, yes. Nice point! Thanks, Kshitij!